deno.json

```diff
@@ -6,11 +6,15 @@
     "format": "deno fmt",
     "lint": "deno lint && deno run -A npm:eslint .",
     "start": "deno serve -A --parallel ./server.production.ts",
-    "typecheck": "deno check '**/*' && deno run -A npm:typescript/tsc",
-    "typegen": "deno types > ./app/deno.d.ts"
+    "typecheck": "deno check --node-modules-dir=auto '**/*'"
```

Comment:
--node-modules-dir=auto is needed here to workaround https://github.com/denoland/deno/issues/26222